hi-ml/primary_deps.yml

```diff
@@ -0,0 +1,16 @@
+# This environment definition contains all packages to run hi-ml and hi-ml-azure development work, building and
+# testing
+name: himl
+channels:
+  - defaults
+  - pytorch
+dependencies:
+  - pip=20.1.1
+  - python=3.7.3
+  - pytorch=1.10.0
+  - cudatoolkit=11.3.1
+  - pip:
+      - -r ../hi-ml-azure/run_requirements.txt
+      - -r run_requirements.txt
+      - -r ../build_requirements.txt
+      - -r ../test_requirements.txt
```

Comment:
how this changes to the environment files are related to the seed features? what's the need of splitting in environment and primary_deps files?

Reply:
It is unrelated. Moving to a separate PR.